EntitiesCollection/EntitiesCollection.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Entities.Entities;
+
+namespace EntitiesCollection;
+
+public class EntitiesCollection
+{
+    private readonly List<IEntity> _entities;
+
+    public EntitiesCollection()
+    {
+        _entities = new List<IEntity>();
+    }
+
+    public void AddEntity(IEntity entity) => _entities.Add(entity);
```

Comment:
what happens if you will add a null to your collection?

Reply:
fixed